content/it/docs/reference/glossary/volume.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Volume
+id: volume
+date: 2022-06-21
+full_link: /it/docs/concepts/storage/volumes/
+short_description: >
+  Una cartella contenente dati, accessibile dai containers all'interno del pod.
+
+aka:
+tags:
+- core-object
+- fundamental
+---
```

Comment:
It seems we are missing the body of the english version of this page

Reply:
Added